libs/pp/include/hpx/pp/strip_parens.hpp

```diff
@@ -52,17 +48,20 @@
  *
```

Comment:
The link above needs to be fixed to: http://boost.2283326.n4.nabble.com/preprocessor-removing-parentheses-td2591973.html#a2591976.

Reply:
Thanks, done!